src/main/java/at/ac/tuwien/kr/alpha/config/CommandLineParser.java

```diff
@@ -0,0 +1,301 @@
+package at.ac.tuwien.kr.alpha.config;
+
+import java.io.ByteArrayOutputStream;
+import java.io.FileInputStream;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Consumer;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import at.ac.tuwien.kr.alpha.common.Predicate;
+
+/**
+ * Parses given argument lists (as passed when Alpha is called from command line) into <code>AlphaConfig</code>s and <code>InputConfig</code>s.
```

Comment:
Please use `{@link AlphaConfig}` instead of `<code>AlphaConfig</code>` and `{@link InputConfig}` instead of `<code>InputConfig</code>`.

Reply:
done